src/indexing_sources/source_pgsql.cpp

```diff
@@ -303,6 +311,28 @@ DWORD CSphSource_PgSQL::SqlColumnLength ( int iIndex )
 	return sph_PQgetlength ( m_pPgResult, m_iPgRow, iIndex );
 }
 
+
+const char * CSphSource_PgSQL::SqlColumnStream( int iIndex, DWORD &iStreamLength )
+{
+	const char * value = SqlColumn( iIndex );
+	size_t iUnpackedLength;
+	iStreamLength = 0;
+
+	if ( !value )
+		return NULL;
+
+	const char * sStream = (const char *) sph_PQunescapeBytea ( (const unsigned char *) value, &iUnpackedLength);
```

Comment:
here is `auto sStream`, or `auto szStream`, as type explicitly casted then, and obvious (CLang will warn about the same explicitly)